packages/server/src/core/parser.rs

```diff
@@ -28,6 +29,16 @@ pub fn parse_copy_paste_data(data: &str) -> Result<Vec<CourseStatus>, AppError>
     let mut sport_courses = Vec::<CourseStatus>::new();
     let mut semester = String::new();
     let mut semester_counter: f32 = 0.0;
+    let should_reverse_credit = CREDIT_RE
+        .captures_iter(
+            data.split_terminator('\n')
+                .filter(|line| COURSE_ID_RE.is_match(line) && !line.contains("ת.ז"))
+                .collect::<Vec<_>>()
+                .join("\n")
+                .as_str(),
+        )
+        .any(|credit| !credit["credit"].ends_with('0') && !credit["credit"].ends_with('5'));
```

Comment:
maybe I am missing something, but you determine if we should reverse credit by checking if there is any course where its decimal value is different than 0 or 5.
So, if the student has only courses with 0, 5, 0.5, 5.0 or 5.5 credits it will result a bug, right?
I know that that's what we did until now but we said it is a hotfix and not a permanent solution.

Reply:
It will, if you have a better solution I am open to suggestions

Reply:
Unfortunately I don't have a better solution.
Let's leave it as it is, but please leave a comment so we would know to think about it in the future.